specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/kubernetesconfiguration.json

```diff
@@ -122,7 +122,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/SourceControlConfiguration"
+              "$ref": "#/definitions/SourceControlConfigurationForCreate"
```

Comment:
request and response body MUST be same. ANy properties which are not settable in the request by end user but present in response can be marked as readONly. it will help in GET-PUT pipeline .

Reply:
For us, it is the other way around.  The user provides a property (configurationProtectedSettings, which is a dictionary of properties) while creating the resource that are not returned in GET.  We want to explicitly let the clients know that this property is not available in PUT & GET responses, as this is sensitive data.  This is why we have two separate definitions for each of these.  Can you suggest if there is any alternative declaration, if this is not acceptable?